allensdk/test/brain_observatory/filtered_running_speed/test_filtered_running_speed_module.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+import os
+import tempfile
+import pathlib
+import pandas as pd
+
+from allensdk.brain_observatory.\
+    filtered_running_speed.multi_stimulus_running_speed import (
+        MultiStimulusRunningSpeed
+    )
+
+DATA_DIR = os.environ.get(
+    "ECEPHYS_PIPELINE_DATA",
+    os.path.join(
+        "/",
+        "allen",
+        "aibs",
+        "informatics",
+        "module_test_data",
+        "ecephys",
+        "filtered_running_speed"
+    ),
+)
+
+NUMBER_OF_VSYNCS = 523198
+NUMBER_OF_NET_ROTATION_ITEMS = 522974
+BEHAVIOR_START = 0
+MAPPING_START = 215999
+REPLAY_START = 307199
+
+
+@pytest.mark.requires_bamboo
+@pytest.fixture(scope="session")
+def multi_stimulus_fixture(tmpdir_factory):
+
+    temp_output_dir = pathlib.Path(
+        tmpdir_factory.mktemp('MultiStimulusRunningSpeedOutput')
+    )
+
+    output_path = tempfile.mkstemp(
+        dir=temp_output_dir,
+        prefix='output_',
+        suffix='.h5')[1]
+
+    output_json = tempfile.mkstemp(
+        dir=temp_output_dir,
+        prefix='output_',
+        suffix='.json')[1]
+
+    mapping_pkl_path = os.path.join(
+        DATA_DIR,
+        '1090803859_553960_20210317_mapping.pkl'
+    )
+
+    behavior_pkl_path = os.path.join(
+        DATA_DIR,
+        '1090803859_553960_20210317_behavior.pkl'
+    )
+
+    replay_pkl_path = os.path.join(
+        DATA_DIR,
+        '1090803859_553960_20210317_replay.pkl'
+    )
+
+    sync_h5_path = os.path.join(
+        DATA_DIR,
+        '1090803859_553960_20210317.sync'
+    )
+
+    args = {
+        'mapping_pkl_path': mapping_pkl_path,
+        'behavior_pkl_path': behavior_pkl_path,
+        'replay_pkl_path': replay_pkl_path,
+        'sync_h5_path': sync_h5_path,
+        'output_json': output_json,
+        'output_path': output_path,
+        'use_lowpass_filter': True,
+        'zscore_threshold': 10.0
+    }
+
+    return MultiStimulusRunningSpeed(
+        args=[],
+        input_data=args
+    )
+
+
+# smoke test
+@pytest.mark.requires_bamboo
+def test_proccessing(multi_stimulus_fixture):
+    try:
+        multi_stimulus_fixture.process()
+    except Exception as exception:
```

Comment:
You shouldn't need this. If a `test_` method in pytest raises an exception, the test will fail.

Reply:
I've updated this.